docker-compose.yml

```diff
@@ -16,7 +16,9 @@ services:
 
   backend:
     networks: [pah]
-    image: codeforpoznan/pah-fm-backend
+    build:
+      context: ./backend
+    #image: codeforpoznan/pah-fm-backend
```

Comment:
As I understand by default we want to pull image from docker hub, not building it locally.

Reply:
yes, will correct that